README.md

```diff
@@ -390,16 +386,13 @@ Please read our [thread_test.rb](https://github.com/rails-sqlserver/tiny_tds/blo
 
 ## Emoji Support 😍
 
-This is possible using FreeTDS version 0.95 or higher. You must use the `use_utf16` login option or add the following config to your `freetds.conf` in either the global section or a specfic dataserver. If you are on Windows, the default location for your conf file will be in `C:\Sites`.
+This is possible. You must use the `use_utf16` login option or add the following config to your `freetds.conf` in either the global section or a specfic dataserver. If you are on Windows, the default location for your conf file will be in `C:\Sites`.
 
 ```ini
 [global]
   use utf-16 = true
 ```
 
-The default is true and since FreeTDS v1.0 would do this as well.
```

Comment:
Can you fold this sentence into the above paragraph? If it's the default then no config file is _required_ to get utf 16 working

Reply:
addressed with bf004345.